feature-libs/my-account/organization/occ/converters/occ-user-group-list-normalizer.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { UserGroup } from '@spartacus/my-account/organization/core';
+import {
+  Converter,
+  Occ,
+  EntitiesModel,
+  ConverterService,
+} from '@spartacus/core';
+import { USER_GROUP_NORMALIZER } from '@spartacus/my-account/organization/core';
+
+@Injectable()
+export class OccUserGroupListNormalizer
+  implements Converter<Occ.OrgUnitUserGroupList, EntitiesModel<UserGroup>> {
+  constructor(private converter: ConverterService) {}
+
+  convert(
+    source: Occ.OrgUnitUserGroupList,
+    target?: EntitiesModel<UserGroup>
+  ): EntitiesModel<UserGroup> {
+    if (target === undefined) {
+      target = {
+        ...(source as any),
+        values: source.orgUnitUserGroups.map((userGroup) => ({
```

Comment:
Placing the logic _inside_ `if (target === undefined)` breaks the idea of plugging multiple normalizers.
IMO it should be:
```typescript
if (target === undefined) {
  target = { ...(source as any) };
}
target.values = /* ... */
```